src/libraries/System.Text.Json/tests/Common/ReferenceHandlerTests/ReferenceHandlerTests.Deserialize.cs

```diff
@@ -243,13 +243,13 @@ public async Task TestJsonPathDoesNotFailOnMultiThreads()
             await Task.WhenAll(tasks);
         }
 
-        private async void TestIdTask()
+        private async Task TestIdTask()
```

Comment:
Removes any `async void` incidents per https://learn.microsoft.com/en-us/archive/msdn-magazine/2013/march/async-await-best-practices-in-asynchronous-programming#avoid-async-void